app/views/reports/user_last_sign_in_at_report.html.erb

```diff
@@ -0,0 +1,49 @@
+<% @colors = ['#F7464A', '#46BFBD', '#FFC870', '#949FB1'] %>
+<div class="row">
+  <div class="col-md-3">
+    <fieldset>
+      <legend>Last login</legend>
+      <ul style="padding-left: 0px; list-style-type: none">
+        <% count = 0 %>
+        <% @counts.each do |key, value| %>
+          <li>
+            <span style="background-color: <%= @colors[count] %>; width: 10px; border: 1px solid <%= @colors[count] %>">
```

Comment:
@SeanLF since it's only the background-color and border-color that are dynamic, consider moving the rest to CSS.